.github/CODEOWNERS

```diff
@@ -12,48 +12,80 @@
 # Samples
 /sdk/samples/                     @rickwinter @chlowell @jhendrixMSFT @richardpark-msft
 
+# AzureSDKOwners: @jhendrixMSFT
+# ServiceLabel: %Azure.Core
 # PRLabel: %Azure.Core
 /sdk/azcore/                  @rickwinter @chlowell @jhendrixMSFT @richardpark-msft
 
+# AzureSDKOwners: @chlowell
+# ServiceLabel: %Azure.Identity
 # PRLabel: %Azure.Identity
 /sdk/azidentity/              @chlowell @jhendrixMSFT @rickwinter @Azure/azure-sdk-write-identity
 
+# AzureSDKOwners: @richardpark-msft
+# ServiceLabel: %OpenAI
 # PRLable: %OpenAI
 /sdk/ai                       @richardpark-msft @jhendrixMSFT
 
+# AzureSDKOwners: @jhendrixMSFT
+# ServiceLabel: %Internal
 # PRLabel: %Internal
 /sdk/internal/                @chlowell @jhendrixMSFT @richardpark-msft @rickwinter
 
+# AzureSDKOwners: @jhendrixMSFT 
+# ServiceLabel: %App Configuration
 # PRLabel: %App Configuration
 /sdk/data/azappconfig/        @antkmsft @jhendrixMSFT @rickwinter
```

Comment:
Do you want to remove Anton from here too while we're changing the file (separate is fine too)?

Reply:
Updated